src/yunohost/domain.py

```diff
@@ -218,3 +218,36 @@ def domain_remove(auth, domain, force=False):
     hook_callback('post_domain_remove', args=[domain])
 
     msignals.display(m18n.n('domain_deleted'), 'success')
+
+
+def domain_generate_dns_configuration(auth, domain):
+    """
+    Generate DNS configuration for a domain
+
+    Keyword argument:
+        domain -- Domain name
+    """
+
+    ip4 = urlopen("http://ip.yunohost.org").read().strip()
+
+    result = "@ 1400 IN A {ip4}\n* 900 IN A {ip4}\n".format(ip4=ip4)
+
+    ip6 = None
+
+    try:
+        ip6 = urlopen("http://ip6.yunohost.org").read().strip()
+    except Exception:
+        pass
+    else:
+        result += "@ 1400 IN A {ip6}\n* 900 IN AAAA {ip6}\n".format(ip6=ip6)
+
+    result += "\n_xmpp-client._tcp 14400 IN SRV 0 5 5222 {domain}.\n_xmpp-server._tcp 14400 IN SRV 0 5 5269 {domain}.\n".format(domain=domain)
```

Comment:
and if I'm reading DNS correctly, the wildcard declaration covers those cases already. But I'm not really an expert in DNS.